### DISCUSSION TITLE
Building arm64 images using macos-latest

### DISCUSSION THREAD

**Discussion by cesarcoatl** [docker/setup-buildx-action#320] [Q&A] [answered]
Hello everyone,

I have also experienced slow runs when building for `linux/arm64` and I was wondering if someone has successfully done so while using `macos-latest`.

Thanks,
cesarcoatl

**Comment by crazy-max** [comment, marked as answer]
Same as https://github.com/docker/setup-buildx-action/discussions/343#discussioncomment-10060474
